lib/CMakeLists.txt

```diff
@@ -19,3 +19,4 @@ add_subdirectory_ifdef(CONFIG_SMF smf)
 add_subdirectory_ifdef(CONFIG_MIN_HEAP min_heap)
```

Comment:
Commit message should at least give a brief overview of libMP similar with the PR description. Keep in mind that once the PR is merged the PR description would be not really useful. People will still look at git log.